cfme/tests/integration/test_auth_manual.py

```diff
@@ -719,97 +719,3 @@ def test_session_timeout():
         title: Verify session timeout works fine for external auth.
     """
     pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_pwd_trailing_whitespace():
-    """
-    Test changing password to one with trailing whitespace.
-
-    Polarion:
-        assignee: jdupuy
-        casecomponent: Auth
-        caseimportance: medium
-        initialEstimate: 1/8h
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_pwd_special_chars():
-    """
-    Test password with special characters.
-
-    Polarion:
-        assignee: jdupuy
-        casecomponent: Auth
-        caseimportance: medium
-        initialEstimate: 1/8h
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_pwd_blank():
-    """
-    Test changing password to a blank one.
-    Test creating user with a blank password.
-
-    Polarion:
-        assignee: jdupuy
-        casecomponent: Auth
-        caseimportance: medium
-        initialEstimate: 1/8h
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_pwd_16_chars():
-    """
-    Password > 16 char
-
-    Polarion:
-        assignee: jdupuy
-        casecomponent: Auth
-        caseimportance: medium
-        initialEstimate: 1/8h
-    """
-    pass
-
-
-@pytest.mark.manual
-@pytest.mark.tier(2)
-def test_login_invalid_user():
```

Comment:
Nice PR ! Is this TC also automated already ?

Reply:
There are tests that cover bad passwords and the like in `test_login.py` but nothing that covers an invalid user. I am not really certain how to determine if a user is invalid and the test case doesn't provide many details. Possibly it's a user that just isn't defined in the DB, which would be easiest enough to automate.

So, I guess I will leave this test case for now and automate it in a separate PR.